debugger/server/debugger_server.go

```diff
@@ -107,16 +109,17 @@ func (ds *DebugServer) Start() (string, error) {
 			}
 		}
 	}()
-	return l.Addr().String(), nil
+	return nil
 }
 
 // Stop stops the server
 func (ds *DebugServer) Stop() {
+	fmt.Printf("stopping debug server")
```

Comment:
We should use `conslogging` and a prefix (eg `WithPrefix("debugger")`) for messages like this, so that it looks a little cleaner.

... though I wonder if the user even needs to know about the debug server. Maybe we don't say anything unless there's an error?

Reply:
whoops, this was some left-over debugging code that slipped in. I'll remove it.